README.md

```diff
@@ -0,0 +1,58 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# SignUp Page
+
+> This project contains a clone of mint.com's signup page.
+
+![screenshot](./project-screenshot.png)
+
+
+## Built With
+
+- HTML
+Designed in Semantic HTML style.
+
+-CSS
+Heavy use of Flexbox
+
```

Comment:
[OPTIONAL] Add a space between - and CSS so markdown understands it to be another point.